src/clickfunnels/main.ts

```diff
@@ -9,11 +14,14 @@ document.addEventListener('DOMContentLoaded', () => {
   const url = new URL(window.location.href);
   const queryParams: Record<string, string> = {};
 
+  analytics.register(urlPathFilter);
```

Comment:
I think this needs to be called before `enableSegment` gets called so that we can make sure the right things are appended to the `.page()` call and anything else that `enableSegment` does automatically internally.

Maybe we could avoid this trouble in the future by instead updating `enableSegment` to take an optional list of plugins as arguments and letting `enableSegment` take care of the correct order of operations internally.

So here in `main.ts` you could just do `enableSegment('YTsllNl2tO9CSJ8OqG7qtz2EW00HElZG', urlPathFilter)` and it'll just work.

Reply:
Looks good to me :-)